stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/ChatDomain.kt

```diff
@@ -589,86 +586,55 @@ public interface ChatDomain {
             this.user = user
         }
 
-        private var database: ChatDatabase? = null
+        private val offlineChatDomainBuilder: OfflineChatDomainBuilder =
+            OfflineChatDomainBuilder(appContext, client, user)
```

Comment:
Shouldn't remove the `user` from this builder?

Reply:
We have to keep old behaviour in new version. Even though this user has been deprecated it has an impact on logic inside `ChatDomainImpl`. This PR shouldn't change any logic, but provide all old without `LiveData` dependency for `ChatDomain`.